Assignment3/Assignment3.py

```diff
@@ -0,0 +1,82 @@
+# Importing necessary libraries
+import sys
+from pyspark import StorageLevel
+from pyspark.sql import SparkSession
+from pyspark.ml import Transformer
+
+# Constructing a transformer
+
+class Rollingaverage(Transformer):
+
+    pass
+
+
+
+def main():
+    # Building the connection
+    spark = SparkSession.builder.config("spark.jars",
+                                        "/Users/nsama6043/Desktop/NewProj/BDA602-ML-/Assignment3/mysql-connector-java-5.1.46/mysql-connector-java-5.1.46.jar") \
+        .master("local").appName("PySpark_MySql_test").config("spark.memory.offHeap.enabled", "true").config(
+        "spark.memory.offHeap.size", "10g").getOrCreate()
+   # storing the game table in a game data frame
+
+    game_query = """
+             SELECT                               
+              game_id,
+              date(local_date) as local_date
+              From game
+              """
+
+    game= spark.read.format("jdbc")\
+        .option("url","jdbc:mysql://localhost:3306/baseball")\
```

Comment:
Instead of passing a query two times. You can create a function for it to get data.

Reply:
Yeah done that!